src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -24,21 +24,23 @@ import {ColourScaleType} from "../../store/colourScales/colourScales";
                 <form novalidate>
                     <div class="row p-0 mb-2">
                         <label for="custom-min-input" class="col col-form-label col-2"><span v-translate="'min'"></span></label>
-                        <div class="col pt-1">
+                        <div class="col pt-1" :style="{paddingRight: 5 + 'px'}">
```

Comment:
Not ideal to use in line style. I'll try to figure out what bootstrap classes to use.